src/store/modules/users.js

```diff
@@ -248,5 +259,8 @@ export const mutations = {
   },
   [types.RECEIVE_USER] (state, { user }) {
     Vue.set(state.entries, user.id, user)
+    if (!state.idList.includes(user.id)) {
```

Comment:
Do we actually need/use the `idList` for users? I left it off when I added the agreements module as we only ever need to get by id.

Reply:
Maybe for the users list? But they might need to get sorted on-the-fly anyways. I'd leave it in for now.. we could open an issue in the future.